tasks/doc.py

```diff
@@ -2,5 +2,6 @@
 
 install_agent: str = f"Install the Agent (default {tool.get_default_agent_install()})."
 agent_version: str = "The version of the Agent for example '7.42.0~rc.1-1' or '6.39.0 (default `latest`)'"
+agent_docker_version: str = "The docker version of the Agent for example '7.45.0-rc.3' (default `latest`)'"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
We should support also agent full path option, useful to test pre-release, pre-rc versions

Reply:
Do you mean supporting the repository path?

Reply:
I think it can be added in a follow PR.

Reply:
Rather the full path to the image, more friendly than using version + repo path, as you can copy and paste the full path from the CI output. Ok for the follow up !